adoc/Private-Registry-Installation.adoc

```diff
@@ -316,6 +316,9 @@ internalTLS:
 ====
 Internal TLS support does not yet cover the internal database and Redis services.
 ====
++
+If {spr} is deployed in k3s, note that unmodified traefik (the default ingress controller bundled with k3s) will not work with automatically generated certificates.
```

Comment:
As there are many certificate configurations maybe its worthy mentioning that it won't work when using internalTLS with automatically generated certificates?

Reply:
Well, this text is in the section that already describes internalTLS, so I think it's clear from the context...

Reply:
indeed! 👍